Makefile

```diff
@@ -321,7 +330,11 @@ prd/embed.html: src/index.mako.html \
 	    --var "translation_fallback_code=$(TRANSLATION_FALLBACK_CODE)" \
 	    --var "default_extent"="$(DEFAULT_EXTENT)" \
 	    --var "default_resolution"="$(DEFAULT_RESOLUTION)" \
-	    --var "public_url=$(PUBLIC_URL)" $< > $@
+	    --var "resolutions"="$(RESOLUTIONS)" \
+	    --var "public_url=$(PUBLIC_URL)" \
+	    --var "default_epsg"="$(DEFAULT_EPSG)" \
+	    --var "default_epsg_extend"="$(DEFAULT_EPSG_EXTEND)" $< > $@
+	${PYTHON_CMD} .build-artefacts/python-venv/bin/mako-render --var "device=embed" --var "mode=prod" --var "version=$(VERSION)" --var "versionslashed=$(VERSION)/" --var "apache_base_path=$(APACHE_BASE_PATH)" --var "api_url=$(API_URL)" --var "default_topic_id=$(DEFAULT_TOPIC_ID)" --var "translation_fallback_code=$(TRANSLATION_FALLBACK_CODE)" --var "default_extent"="$(DEFAULT_EXTENT)" --var "default_resolution"="$(DEFAULT_RESOLUTION)" --var "resolutions"="$(RESOLUTIONS)" --var "public_url=$(PUBLIC_URL)" $< > $@
```

Comment:
Indeed. I guess I just tested with `make dev` and not `make prod`. Commit updated.